src/pad/README.md

```diff
@@ -1,21 +1,282 @@
+| `-ring_index` | (Optional) Used to specify the index of the ring in case of multiple rings. |
+
+#### Examples
+```
+make_io_sites -horizontal_site IOSITE_H -vertical_site IOSITE_V -corner_site IOSITE_C -offset 35
+make_io_sites -horizontal_site IOSITE_H -vertical_site IOSITE_V -corner_site IOSITE_C -offset 35 -rotation R180
+```
+
+### Creating False IO Sites
+
+If the library does not contain sites for the IO cells, the following command can be used to add them.
+This should not be used unless the sites are not in the library.
+```
+make_fake_io_site -name name \
+                  -width width \
+                  -height height
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-name` | Name of the site. |
+| `-width` | Width of the site in um. |
+| `-height` | Height of the site in um. |
+
+#### Examples
+```
+make_fake_io_site -name IO_HSITE -width 1 -height 204
+make_fake_io_site -name IO_VSITE -width 1 -height 200
+make_fake_io_site -name IO_CSITE -width 200 -height 204
+```
+
+### Remove IO Rows
+
+When the padring is complete, the following command can remove the IO rows to avoid causing confusion with the other tools
+```
+remove_io_rows
+```
+
+### Placing Pads
+
+To place a pad into the pad ring
+```
+place_pad -row row_name \
+          -location offset \
+          -mirror \
+          [-master master] \
+          name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-row` | Name of the row to place the pad into, examples include: IO_NORTH, IO_SOUTH, IO_WEST, IO_EAST, IO_NORTH_0, IO_NORTH_1 |
+| `-location` | Offset from the bottom left chip edge to place the pad at. |
+| `-mirror` | Specifies if the pad should be mirrored. |
+| `-master` | Name of the instance master if the instance needs to be created. |
+| `name` | Name of the instance |
+
+#### Examples
+```
+place_pad -row IO_SOUTH -location 280.0 {u_clk.u_in}
+place_pad -row IO_SOUTH -location 360.0 -mirror {u_reset.u_in}
+place_pad -master sky130_fd_io__top_ground_hvc_wpad -row IO_SOUTH -location 439.5 {u_vzz_0}
+place_pad -master sky130_fd_io__top_power_hvc_wpad -row IO_SOUTH -location 517.5 {u_v18_0}
+```
+
+### Placing corners
+
+To place the corner cells
+```
+place_corners master \
+              [-ring_index index]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-ring_index` | (Optional) Used to specify the index of the ring in case of multiple rings. |
+| `master` | Name of the master for the corners. |
+
+#### Examples
+```
+place_corners sky130_fd_io__corner_bus_overlay
+```
+
+### Placing IO Filler Cells
+
+To place the IO filler cells
+```
+place_io_fill -row row_name \
+              masters
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-row` | Name of the row to place the pad into, examples include: IO_NORTH, IO_SOUTH, IO_WEST, IO_EAST, IO_NORTH_0, IO_NORTH_1 |
+| `masters` | Names of the masters for the IO filler cells. |
```

Comment:
I can certainly add something to allow overlaps. It's usually not all filler cells that are overlapable, so it might need to be two different calls (one without allowing overlaps, but allowing gaps) followed by a second call with the overlapping ones? or an additional flag to indicate which cells allow for overlaps? (I'm partial to the second option), which do you think makes sense?